### PR TITLE
File index.html  does not exist in INSTALL.txt

### LINKED ISSUE #21463: INSTALL.txt instructions are wrong
#### Is your feature request related to a problem?

Install.txt instructions are wrong, index.html don't exist 

#### Describe the solution you'd like

Install.txt instructions are wrong

#### Alternatives you've considered

Remove text or Change by index.php

### PR DESCRIPTION
**Pull request by ghost**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop 
| Description?  | index.html  does not exist
| Type?         | improvement
| Category?     |  CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes https://github.com/PrestaShop/PrestaShop/issues/21463
| How to test?  | Try the url index.html

To complete this one : https://github.com/PrestaShop/PrestaShop/pull/21453/

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/21464)

### CODE REVIEW

**Review by Progi1984 — DISMISSED** [review]

**Review by matks — DISMISSED** [review]

File: INSTALL.txt [hunk 1]
```diff
@@ -63,7 +63,7 @@ Here is the content of this archive:
 From there on, follow these instructions:
 
 1. Upload at least index.php and prestashop.zip on your web server.
-2. From your web browser, go to the folder where index.php and prestashop.zip have been uploaded and browse index.html. The Zip archive should unzip automatically.
+2. From your web browser, go to the folder where index.php and prestashop.zip have been uploaded. The Zip archive should unzip automatically.
```
Comment (matks) [inline INSTALL.txt]:
```suggestion
2. From your web browser, go to the folder where index.php and prestashop.zip have been uploaded and browse index.php. The Zip archive should unzip automatically.
```

index.html => index.php

On 2nd thought ... 🤔 how is it ? better or not ? I'm 50/50 on it

Comment (ghost) [reply INSTALL.txt]: i say the same in the issue https://github.com/PrestaShop/PrestaShop/issues/21463

Heads or tails ? :rofl:

Comment (matks) [reply INSTALL.txt]: `rand(1,2)` => my php returned 2

**Comment by ghost** [comment]
you win ;'(

**Review by Progi1984 — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks @okom3pom

**Pull request merged**